roles/cinder-common/templates/etc/cinder/cinder.conf

```diff
@@ -45,16 +38,24 @@ backup_driver = cinder.backup.drivers.swift
 backup_swift_url = http://{{ endpoints.swift.haproxy_vip }}:{{ endpoints.swift.port.cinder_backup }}/v1/
 {% endif -%}
 
+{% if not ceph.enabled|bool -%}
+default_volume_type = LVM
+enabled_backends = lvm
+[lvm]
+volume_backend_name = lvm
+volume_driver = cinder.volume.drivers.lvm.LVMVolumeDriver
+iscsi_helper=tgtadm
+iscsi_ip_address = {{ primary_ip }}
+volume_group = cinder-volumes
+use_forwarded_for = true
+{% endif -%}
+
 {% if ceph.enabled|bool -%}
 {% if cinder.common_volume_name -%}
 host = ceph
 {% endif -%}
 
-{% if cinder.default_volume_type | default(false) -%}
```

Comment:
We allowed operator to force define default volume_type via cinder.default_volume_type .. Removing below lines will break that

Example: Customer had all ssd ceph cluster and later decides to add hybrid ceph. If operator does not override default volume_type to CEPH_SSD than by default CEPH_Hybrid will be default volume_type and customer default behavior will change for cinder create ..